src/content/h1/sources/animation-data/readme.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Animation data
+workflowName: Animation data
+stub: true
+keywords:
+  - jma
+  - animation
+thanks:
+  - to: General_101
+    for: Documentation on animation types
+  - to: ODX
+    for: JMO frames
+---
+The [HEK][] uses several file types as intermediate representations of animation data. These files provide a common target for exporters like [Bluestreak][] and [H2V Blender Animation Exporter][h2v-blender-animation-exporter], and can then be converted to [model_animations][] tags by [Tool][] for use in Halo.
+
+# JMA (Base)
+This is what a lot of animations will end up as. It is a default animation type that stores data necessary for movement, such as a [biped][] walking animation.
+
+# JMM (Base)
+...
```

Comment:
JMM is for movement, can we check to make sure this description is right?

Reply:
This is from looking at Bluestreak's source. JMM is:

* animation type == 0
* frame info type == 0
* world relative == false

The animation type enum is Base, Overlay, Replacement.

Reply:
Pool has help text on this
https://github.com/Sigmmma/hek_pool/blob/master/hek_pool/help_text.py#L82

Reply:
That's helpful info, thanks. I (or someone) can add that animations information in another pull request.